First/ViewController.swift

```diff
@@ -8,132 +8,116 @@
 import UIKit
 
 class ViewController: UIViewController {
-    var userName: UITextField!
-    var password: UITextField!
-    var legalTerms: UILabel!
-    var login: UIButton!
-    var trouble: UIButton!
-    let daymix = UIImage(named: "DaymixLogo.png")
-    var logo: UIImageView!
-    var blackBackground: UIImageView!
+    private let kLogoWidth: CGFloat = 250
+    private let kLogoHeight: CGFloat = 100
+    private let kLeftMargin: CGFloat = 10
+    private let kRightMargin: CGFloat = -10
```

Comment:
Bardzo ładnie to wygląda, tylko te - stosujemy już w kodzie, a nie przy deklaracji zmiennych